Functions/src/main/java/com/epam/engx/cleancode/functions/task1/RegisterAccountAction.java

```diff
@@ -10,21 +10,26 @@
 
 public class RegisterAccountAction {
 
-
+    private RegisterAccountAction registerAccountAction;
     private PasswordChecker passwordChecker;
     private AccountManager accountManager;
 
     public void register(Account account) {
-        if (account.getName().length() <= 5){
-            throw new WrongAccountNameException();
-        }
-        String password = account.getPassword();
-        if (password.length() <= 8) {
-            if (passwordChecker.validate(password) != OK) {
-                throw new WrongPasswordException();
-            }
+        try {
+            isValidName(account);
+            isValidPassword(account.getPassword());
+        } catch (WrongAccountNameException en) {
+            System.out.println("invalid name");
+            en.printStackTrace();
+        } catch (WrongPasswordException ep) {
+            System.out.println("invalid password");
+            ep.printStackTrace();
```

Comment:
The method changed completely. Now all muted and printing instead of throwing exceptions to super layer and indicate problems as expected